packer/build_deb_image.sh

```diff
@@ -320,7 +321,8 @@ set -x
   -var-file="$DIR"/variables.json \
   -var install_args="$INSTALL_ARGS" \
   -var ssh_username="$SSH_USERNAME" \
-  -var scylla_version="$SCYLLA_VERSION" \
+  -var scylla_full_version="$SCYLLA_FULL_VERSION" \
+  -var version="$VERSION" \
```

Comment:
/version/scylla_version/ - not critical but to make it clear like the other tools on the next rows

Reply:
I missed this comment , will send a follow up PR

Reply:
https://github.com/scylladb/scylla-machine-image/pull/384